src/Integration.Vsix/SonarLintDaemon/DaemonIssueAdapter.cs

```diff
@@ -28,15 +29,15 @@ internal class DaemonIssueAdapter : IFilterableIssue
         private readonly string wholeLineText;
         private readonly string lineHash;
 
-        public DaemonIssueAdapter(Sonarlint.Issue sonarLintIssue, string wholeLineText, string lineHash)
+        public DaemonIssueAdapter(IAnalysisIssue sonarLintIssue, string wholeLineText, string lineHash)
```

Comment:
This class is not longer specific to the daemon. It should be possible now to move it to `Core` and rename it.
There are few other interfaces/classes that can be moved. I'll do that in a separate PR.